protoc-gen-grpc-gateway/descriptor/types.go

```diff
@@ -227,10 +234,10 @@ func (p Parameter) ConvertFuncExpr() (string, error) {
 	return conv, nil
 }
 
-// Body describes a http requtest body to be sent to the method.
+// Body describes a http request(response) body to be sent to the method(client).
```

Comment:
How about the bash syntax:

```
// Body describes a http {request,response} body to be sent to the {method,client}.
```

or the regex syntax:

```
// Body describes a http (request|response) body to be sent to the (method|client).
```

Maybe expand the comment a bit, and mention this is used in `body` and `response_body` options in `google.api.HttpRule`?